### DISCUSSION TITLE
failed to extract shortcode: template for shortcode "notice" not found

### DISCUSSION THREAD

**Discussion by CodeinLR** [McShelby/hugo-theme-relearn#276] [General]
hi 
i have this error with all shortcode
content/_index.en.md:9:1": failed to extract shortcode: template for shortcode "notice" not found
could you help  me please ?

**Comment by McShelby** [comment]
Your installation may be broken. See:
https://mcshelby.github.io/hugo-theme-relearn/basics/installation/
https://gohugo.io/getting-started/quick-start/
